config/development.config.php.dist

```diff
@@ -1,4 +1,6 @@
 <?php
+
+declare(strict_types=1);
```

Comment:
we always did comments first, declare second

Reply:
and again, c&p

all fixed with latest push